conformance/adequacy/check_published_numbers.py

```
@@ -115,6 +118,135 @@
+        return UNREACHABLE, []
+    for path in paths:
+        measured_kind = git_tree_entry_kind(repo, commit, path)
+        head_kind = git_tree_entry_kind(repo, "HEAD", path)
+        if measured_kind is None or head_kind is None:
+            return UNREACHABLE, []
+        if "nonregular" in (measured_kind, head_kind):
+            return MALFORMED, [path]
+        if measured_kind == "absent" and head_kind == "absent":
+            return MALFORMED, [path]
+    try:
+        moved = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)